### PR TITLE
XIAS, overview, and UABMC section updates

### PR DESCRIPTION
**Pull request by bjoyce3**
Updated XIAS account process and added UABMC section. Attempting to fix https://github.com/uabrc/uabrc.github.io/issues/83

### CODE REVIEW

File: docs/account_management/gitlab_user.md [hunk 1]
```diff
@@ -1,31 +1,57 @@
-# UAB GitLab Registration Instructions
+# UAB GitLab Overview
+
+Welcome to UAB GitLab! This is a UAB-specific [GitLab](https://about.gitlab.com/). GitLab is similar to [GitHub](https://github.com/), but hosted here at UAB on secure servers. 
+
+### Researcher GitLab Use Cases
```
Comment (mdefende) [inline docs/account_management/gitlab_user.md:5]: Heading levels are strange, right? Looks like It goes from the page title  on line 5 (single #)to a subheading level (###) on line 5.

File: docs/account_management/gitlab_user.md [hunk 2]
```diff
@@ -1,31 +1,57 @@
...
+     - Software versions can be recorded, virtual environments can be documented, and containers can be recorded to help future-proof analyses.
+   - Collaboration
+     - GitLab is a central place to create code, edit, and track needed code changes (issues) with your lab and collaborators.
+     - Multiple people can use, modify, and merge changes in code while communicating with the broader team all along the way.
+   - Security
+     - Unlimited private repositories for internal code projects.
+     - Set behind UAB authentication.
+
+### Software Developers (and Researchers!) GitLab Use Cases
+UAB GitLab is useful for software developers. It is a single application for the entire software development lifecycle. From project planning and source code management to continuous integration (CI) and continuous deployment (CD), monitoring, and security.
```
Comment (mdefende) [inline docs/account_management/gitlab_user.md:20]: I think it's best practice to have a line break between the subheading and the text body

File: docs/account_management/gitlab_user.md [hunk 3]
```diff
@@ -1,31 +1,57 @@
...
 
 Our GitLab instance may be found at <https://gitlab.rc.uab.edu>.
 
-## UAB Affiliated Users
+# UAB GitLab Registration
```
Comment (mdefende) [inline docs/account_management/gitlab_user.md:24]: Only one top-level heading is "allowed" in a document. Shouldbe changed to ##

Comment (mdefende) [reply docs/account_management/gitlab_user.md:24]: The following subheadings should also have an extra # added to them to keep the organization consistent

**Review by mdefende — CHANGES_REQUESTED** [review]
A few headings are not formatted correctly, I think

File: docs/account_management/gitlab_user.md (hunk 1 above)
Comment (bjoyce3) [reply docs/account_management/gitlab_user.md:5]: Yeah should definitely be two #'s

**Comment by bjoyce3** [comment]
I believe I fixed those requests. Can't seem to figure out how to address that in the GitHub interface. Also this PR partials solves https://github.com/uabrc/uabrc.github.io/issues/38

File: docs/account_management/gitlab_user.md (hunk 2 above)
Comment (bjoyce3) [reply docs/account_management/gitlab_user.md:22]: Got it. All the white space messes with me, but I'll do that for the future.

**Review by mdefende — APPROVED** [review]

**Pull request merged**